### PR TITLE
Updated these files to support latest MongoDB version

### PR DESCRIPTION
**Pull request by PratyushRajShukla**

### CODE REVIEW

File: config/db.js [hunk 1]
```diff
@@ -4,9 +4,9 @@ const db = config.get('mongoURI');
 
 const connectDB = async () =>{
     try{
-        await mongoose.connect(db,{
+        await mongoose.connect(db, {
             useNewUrlParser: true,
-            useCreateIndex: true
```
Comment (deeprajjha1) [inline config/db.js:9]: Why did you remove useCreateIndex flag?

Comment (PratyushRajShukla) [reply config/db.js:9]: useNewUrlParser, useUnifiedTopology, useFindAndModify, and useCreateIndex are no longer supported options. Mongoose 6 always behaves as if useNewUrlParser, useUnifiedTopology, and useCreateIndex are true, and useFindAndModify is false. it was showing an error cause of this

Comment (deeprajjha1) [reply config/db.js:9]: Ok. looks good to me

**Review by deeprajjha1 — APPROVED** [review]
Looks good to me
